test/deploy/kubernetes/deploy.sh

```diff
@@ -3,6 +3,10 @@ BASEPATH=$(dirname $0)
 MAXNODES=$(kubectl get nodes | awk -v col=1 '{print $col}' | tail -n+2 | wc -l)
 NAMESPACE="cortx"
 
+# Calling 3rd party component deployment scrip
+
+sh 3rd_party_services/deploy-3rd-party-comp.sh
```

Comment:
We should be able to directly execute shell script without `sh`

Reply:
file permission changed and script call updated